scipy/stats/tests/test_stats.py

```diff
@@ -9596,6 +9630,12 @@ def test_nan_policy(self, xp):
         ref = xp.var(x[~mask])
         xp_assert_close(res, ref)
 
+    @skip_xp_backends(eager_only=True)
+    def test_nan_policy_warns(self, xp):
+        x = xp.arange(10.)
+        mask = (x == 3)
+        x = xp.where(mask, xp.nan, x)
```

Comment:
NBD, but I saw you did it above.
```suggestion
        x = xp.where(x == 3, xp.nan, x)
```